Signal/src/call/CallService.swift

```diff
@@ -453,12 +472,13 @@ protocol CallServiceObserver: class {
         call = newCall
 
         let backgroundTask = UIApplication.shared.beginBackgroundTask {
-            let timeout = CallError.timeout(description: "background task time ran out before call connected.", call: newCall)
```

Comment:
As above, I think we want to continue to specify *which* call has timed out, lest we clobber a subsequent call.